vaadin-testbench-core/src/main/java/com/vaadin/karibu/Karibu5Test.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (C) 2020 Vaadin Ltd
+ *
+ * This program is available under Commercial Vaadin Developer License
+ * 4.0 (CVDLv4).
+ *
+ *
+ * For the full License, see <https://vaadin.com/license/cvdl-4.0>.
+ */
+package com.vaadin.karibu;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+
+/**
+ * Base JUnit 5 class for UI unit tests.
+ * 
+ * Subclasses should typically restrict classpath scanning to a specific package
+ * for faster bootstrap by overriding {@link #scanPackage()} method.
+ */
+public abstract class Karibu5Test extends BaseKaribuTest {
```

Comment:
Again in the naming of the abstract I would perhaps not use Karibu in the name but leave it as an implementation detail.

Also there is a target of going to junit5 as default so I would have this be the base name `UnitTest`? and have the junit4 version have the version in the name `Unit4Test`

Reply:
Same for the test names and packages. to not use karibu.

Reply:
What about `UIUnitTest`? It may better explain the purpose of the base class.